### PR TITLE
Mob 4499 post release 1 10 0 [MASTER]

### PR DESCRIPTION
**Pull request by dianakarh**
## JIRA
[MOB-4499](https://karhoo.atlassian.net/browse/MOB-4499)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooUISDK.xcodeproj/xcshareddata/xcschemes/KarhooUISDK.xcscheme [hunk 1]
```diff
@@ -21,7 +21,7 @@
             </BuildableReference>
          </BuildActionEntry>
          <BuildActionEntry
-            buildForTesting = "NO"
+            buildForTesting = "YES"
```
Comment (mionescu95) [inline KarhooUISDK.xcodeproj/xcshareddata/xcschemes/KarhooUISDK.xcscheme:24]: build for testing?

File: KarhooUISDK/CommonUI/LoadingControls/LoadingImageView/LoadingImageView.swift [hunk 2]
```diff
@@ -7,6 +7,7 @@
 //
 
 import UIKit
+import SwiftUI
```
Comment (mionescu95) [inline KarhooUISDK/CommonUI/LoadingControls/LoadingImageView/LoadingImageView.swift:10]: is it useful?

**Review by mionescu95 — DISMISSED** [review]
a lot of png's were added, is it ok?

File: KarhooUISDK/CommonUI/LoadingControls/LoadingImageView/LoadingImageView.swift (hunk 2 above)
Comment (dianakarh) [reply KarhooUISDK/CommonUI/LoadingControls/LoadingImageView/LoadingImageView.swift:10]: Leftover. Cleaned it up.

File: KarhooUISDK.xcodeproj/xcshareddata/xcschemes/KarhooUISDK.xcscheme (hunk 1 above)
Comment (dianakarh) [reply KarhooUISDK.xcodeproj/xcshareddata/xcschemes/KarhooUISDK.xcscheme:24]: It's related to the Client app, should be safe to merge with no consequences to the DP integrations.

**Review by mateid-kr — APPROVED** [review]

**Review by mionescu95 — APPROVED** [review]

**Pull request merged**